src/validators/user-info-request-validator.ts

```diff
@@ -0,0 +1,68 @@
+import { UserInfoRequestError } from "../errors/user-info-request-error";
+import { IncomingHttpHeaders } from "http";
+import { signedJwtValidator } from "./signed-jwt-validator";
+import { Config } from "../config";
+import { logger } from "../logger";
+
+const AuthorisationHeaderKey: string = "authorization";
+const ValidScopes: string[] = ["openid", "email", "phone"];
+
+export const userInfoRequestValidator = async (
+  userInfoRequestHeaders: IncomingHttpHeaders
+): Promise<
+  | { valid: true }
+  | {
+      valid: false;
+      error: UserInfoRequestError;
+    }
+> => {
+  for (const key of Object.keys(userInfoRequestHeaders)) {
+    userInfoRequestHeaders[key.toLowerCase()] = userInfoRequestHeaders[key];
+  }
+  const accessToken = userInfoRequestHeaders[AuthorisationHeaderKey];
```

Comment:
I think Ethan was concerned the request header might be "Authorization" instead of "authorization". Might be nice to move it out into a helper function.

Reply:
It looks like express handles case sensitivity, so the above is fine.